src/Docs/Session/Create/SdkConfig.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\Docs\Session\Create;
+
+use JsonSerializable;
+
+class SdkConfig implements JsonSerializable
+{
+
+    /**
+     * @var string|null
+     */
+    private $allowedCaptureMethods;
+
+    /**
+     * @var string|null
+     */
+    private $primaryColour;
+
+    /**
+     * @var string|null
+     */
+    private $secondaryColour;
+
+    /**
+     * @var string|null
+     */
+    private $fontColour;
+
+    /**
+     * @var string|null
+     */
+    private $locale;
+
+    /**
+     * @var string|null
+     */
+    private $presetIssuingCountry;
+
+    /**
+     * @var string|null
+     */
+    private $successUrl;
+
+    /**
+     * @var string|null
+     */
+    private $errorUrl;
+
+    public function __construct(
+        ?string $allowedCaptureMethods,
+        ?string $primaryColour,
+        ?string $secondaryColour,
+        ?string $fontColour,
+        ?string $locale,
+        ?string $presetIssuingCountry,
+        ?string $successUrl,
+        ?string $errorUrl
+    ) {
+        $this->allowedCaptureMethods = $allowedCaptureMethods;
+        $this->primaryColour = $primaryColour;
+        $this->secondaryColour = $secondaryColour;
+        $this->fontColour = $fontColour;
+        $this->locale = $locale;
+        $this->presetIssuingCountry = $presetIssuingCountry;
+        $this->successUrl = $successUrl;
+        $this->errorUrl = $errorUrl;
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function jsonSerialize(): array
+    {
+        return [
+            'allowed_capture_methods' => $this->getAllowedCaptureMethods(),
+            'primary_colour' => $this->getPrimaryColour(),
+            'secondary_colour' => $this->getSecondaryColour(),
+            'font_colour' => $this->getFontColour(),
+            'locale' => $this->getLocale(),
+            'preset_issuing_country' => $this->getPresetIssuingCountry(),
+            'success_url' => $this->getSuccessUrl(),
+            'error_url' => $this->getErrorUrl(),
+        ];
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getAllowedCaptureMethods(): ?string
```

Comment:
We could cut down the size of the `Yoti\Docs\Session\Create\*` classes by removing the public getters, as they only need to serialise to JSON _(it would also cut down the number of unit tests required to test these classes)_. This is only a suggestion, happy to keep them if we think they'll be useful in future.

Reply:
If we can't think of a use case for exposing them, I think we should remove them. We can always add the getters at a later stage